### PR TITLE
Add Bloomberg/ForePaaS recap blog

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-fest-2023/TrinoFest2023LetItSnow.pdf">
+  Check out the slides!
+</a>
+
+For those unfamiliar, Snowflake is a cloud-based data warehousing and analytics
+platform. It offers a great combination of scale, flexibility, and performance,
+with the downside of being a proprietary software that is vendor-locked, and in
+order to use Snowflake, you must go through Snowflake, Inc. ForePaaS is a
+customer of Snowflake's, but they also store data in many other formats and
```
Comment (mosabua) [inline _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md:31]: Is that true .. I thought the offer a solution for their customers.

File: _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+systems, and they rely on Trino to run their analytics. With no Snowflake
+connector in Trino, this meant that while they could run analytics and queries
+on most of their data, Trino had a blind spot, and they needed to develop a
+Snowflake connector in order to see and query 100% of their data. Bloomberg was
+in a similar boat, having data in Snowflake, using Trino for analytics, and
+needing a way to join those two together. Naturally, ForePaaS and Bloomberg
```
Comment (mosabua) [inline _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md:37]: Naturally?? Not so much ..

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+For those unfamiliar, Snowflake is a cloud-based data warehousing and analytics
+platform. It offers a great combination of scale, flexibility, and performance,
+with the downside of being a proprietary software that is vendor-locked, and in
+order to use Snowflake, you must go through Snowflake, Inc. ForePaaS is a
+customer of Snowflake's, but they also store data in many other formats and
+systems, and they rely on Trino to run their analytics. With no Snowflake
+connector in Trino, this meant that while they could run analytics and queries
+on most of their data, Trino had a blind spot, and they needed to develop a
+Snowflake connector in order to see and query 100% of their data. Bloomberg was
```
Comment (ryandoowan) [inline _posts/2023-07-12-trino-fest-2023-let-it-snow-recap.md:35]: This sentence feels like it could be separated into two sentences

**Review by mosabua — APPROVED** [review]
Looks good to me now.

**Review by ryandoowan — APPROVED** [review]

**Pull request merged**